docs/source/user_guide/pipelines.ipynb

```diff
@@ -401,8 +401,8 @@
     "# This string can then be pasted into a separate window and run, although since the pipeline has custom component `MyDropNullColumns`, \n",
     "#      the code for that component must also be included\n",
     "\n",
-    "from evalml.demos import load_fraud\n",
-    "X, y = load_fraud()\n",
+    "from evalml.demos import load_breast_cancer\n",
```

Comment:
Was there a reason why this was changed? Run time?

Reply:
Not run time! Since I removed the date time featurizer, we can't use fraud because there's a datetime column. An alternative is to drop the datetime column and not switch datasets but I felt that could distract from the code gen.

Reply:
Got it, thanks!